src/App.vue

```diff
@@ -76,7 +76,7 @@
 
     <!-- Display WebChat for SP/GP registrations only -->
     <WebChat
-      v-if="getFilingType === FilingTypes.REGISTRATION && enableOldWebchat"
+      v-if="enableOldWebchat && getFilingType === FilingTypes.REGISTRATION"
```

Comment:
Yes, this looks cleaner, but for the record, both conditions need to be evaluated so it's not any faster. Conversely, some "or" conditions can be optimized.